packages/react-components/src/collection/src/useCollection.ts

```diff
@@ -14,14 +14,14 @@ export interface CollectionItem {
     content: ElementType | ReactElement[];
     props: Record<string, any>,
     tooltip?: {
-        props: Record<string, any>,
-        content: ReactElement
+        props: Record<string, any>;
+        content: ReactElement;
     },
 }
 
 export interface CollectionSection extends CollectionItem {
     type: NodeType.section;
-    items: CollectionItem[]
+    items?: CollectionItem[]
```

Comment:
add a ";" ?

Why do we use a ";" instead of a "," ? I personally keep making the mistake of using "," and it seems accepted since no compiler errors and we don't have any linting rules on this.

- Spectrum is using ","
- Chakra doesn't use any character

I would personally go for using a "," because an interface syntax is similar to an object literal so it feels natural.

Reply:
I found a eslint rule that ensures this. I'll add it after the PRs.

Typescript historically only allowed semi colon. Since TS 1.6, they support a colon or nothing as the delimiter.

TS still uses semicolon in all their examples, and its the most common delimiter i've seen. 
It's also the default in the eslint rule, specifying its the preferred synthax
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/member-delimiter-style.md

Reply:
All good then!